content/chainguard/chainguard-images/getting-started/getting-started-istio.md

```diff
@@ -0,0 +1,144 @@
+---
+title: "Getting Started with the Chainguard Istio Images"
+type: "article"
+linktitle: "Istio"
+description: "Tutorial on how to get started with the Chainguard Istio Images"
+date: 2023-12-14T00:00:00+00:00
+lastmod: 2023-09-22T11:07:52+02:00
+tags: ["Chainguard Images", "Products"]
+draft: false
+images: []
+menu:
+  docs:
+    parent: "getting-started"
+weight: 610
+toc: true
+---
+
+[Istio](https://istio.io) extends Kubernetes to establish a programmable,
+application-aware network using the powerful Envoy service proxy. Working with
+both Kubernetes and traditional workloads, Istio brings standard, universal
+traffic management, telemetry, and security to complex deployments. Chainguard
+offers a set of minimal, security-hardened Istio images, built on top the Wolfi
+OS.
+
+We will demonstrate how to get started with the Chainguard Istio images on an
+example KinD cluster. To get started, you'll need docker, KinD, kubectl, and istioctl
+installed.
+
+* [Docker](https://docs.docker.com/get-docker/)
+* [KinD](https://kind.sigs.k8s.io/docs/user/quick-start/)
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+* [istioctl](https://istio.io/latest/docs/setup/getting-started/#download)
+
+
+{{< details "What is Wolfi" >}}
+{{< blurb/wolfi >}}
+{{< /details >}}
+
+{{< details "Chainguard Images" >}}
+{{< blurb/images >}}
+{{< /details >}}
+
+## Start up a KinD cluster
+
+First, we'll start up a KinD cluster to install Istio
+```
+kind create cluster
+```
+
+You should see output similar to the following:
+```
+❯ kind create cluster
+Creating cluster "kind" ...
+ ✓ Ensuring node image (kindest/node:v1.27.3) 🖼 
+ ✓ Preparing nodes 📦  
+ ✓ Writing configuration 📜 
+ ✓ Starting control-plane 🕹️ 
+ ✓ Installing CNI 🔌 
+ ✓ Installing StorageClass 💾 
+Set kubectl context to "kind-kind"
+You can now use your cluster with:
+
+kubectl cluster-info --context kind-kind
+
+Thanks for using kind! 😊
+```
+
+## Install Istio using Chainguard Images
+
+We will be using the `istioctl` command to install Istio. In order to use the
+Chainguard images, we will need to set these following values:
+- `hub = cgr.dev/chainguard`
+- `tag = latest`
+- `values.pilot.image = istio-pilot`
+- `values.global.proxy.image = istio-proxy`
+- `values.global.proxy_init.image = istio-proxy`
+
+Those translate to the following `istioctl` command:
+```
+istioctl install --set tag=latest --set hub=cgr.dev/chainguard \
+  --set values.pilot.image=istio-pilot \
+  --set values.global.proxy.image=istio-proxy \
+  --set values.global.proxy_init.image=istio-proxy
+```
+
+## Stand up a Gateway and a VirtualService 
+
+To see the Istio installation in action, we will create two Istio resources:
```

Comment:
```suggestion
To test the Istio installation, we will create two Istio resources:
```
I try to avoid using phrases like "Let's see" or "take a look at" or similar - I was taught to assume that every doc will be read by someone who uses a screen reader and maybe can't look or see.